app/utils/theme.js

```diff
@@ -31,6 +31,9 @@ export function setNavigatorStyles(componentId, theme) {
             },
             leftButtonColor: theme.sidebarHeaderTextColor,
             rightButtonColor: theme.sidebarHeaderTextColor,
+            backButton: {
```

Comment:
this should be optional, as setting this on Android was causing issues in the settings screen where the button was being replaced with a normal back button

Reply:
alright, lets just focus on removing the invalid component Ids and we are good to go